src/buffered-process.coffee

```diff
@@ -159,6 +160,27 @@ class BufferedProcess
           process.kill(pid)
       @killProcess()
 
+  # Kill all child processes spawned by a spawned child process
+  #
+  # Uses ps-tree, so it works on *nix systems only
+  killOnNix: ->
+    return unless @process?
+
+    self = this
+    parentPid = @process.pid
+    psTree(parentPid, (err, children) ->
+      [parentPid].concat(
+        children.map((p) ->
+          return p.PID
+        )
+      ).forEach((tpid) ->
```

Comment:
@thomasjo both of these seem to break the functionality introduced, as the array isn't returned by the (err, children) callback function